charts/linkerd2-cni/Chart.yaml

```diff
@@ -9,4 +9,4 @@ description: |
 kubeVersion: ">=1.21.0-0"
 icon: https://linkerd.io/images/logo-only-200h.png
 name: "linkerd2-cni"
-version: 30.11.0
+version: 30.11.1-edge
```

Comment:
We always bump the minor and reset the patch for the first edge after a stable (to make room for point releases)

Reply:
ditto throughout

Reply:
That's what I initially did (based on the release notes), however @alpeb pointed out that we should instead bump the patch and leave the minor as is. https://github.com/linkerd/linkerd2/pull/11350#discussion_r1318776573